create_gz_vendor_pkg/templates/CMakeLists.txt.jinja

```diff
@@ -16,6 +16,17 @@ set(LIB_NAME_COMP_PREFIX ${LIB_NAME})
 set(LIB_NAME_FULL ${LIB_NAME}${LIB_VER_MAJOR})
 set(LIB_VER ${LIB_VER_MAJOR}.${LIB_VER_MINOR}.${LIB_VER_PATCH})
 
+option(VENDOR_FROM_LIB_VCS_REF
+  "Vendor from a VCS ref instead of tag. Uses the value in LIB_VCS_REF if specified or main otherwise"
+  OFF)
+if(NOT VENDOR_FROM_LIB_VCS_REF)
+  set(LIB_VCS_VER ${GITHUB_NAME}${LIB_VER_MAJOR}_${LIB_VER}${LIB_VER_SUFFIX})
+elseif(LIB_VCS_REF)
```

Comment:
Is `LIB_VCS_REF` supposed to be set somewhere?

Reply:
you can use `--cmake-args -DVENDOR_FROM_LIB_VCS_REF=ON` with no arguments to use the default value of `main` (except for gz-tools2), or you can specify the VCS ref to use as well with `--cmake-args -DVENDOR_FROM_LIB_VCS_REF=ON -DLIB_VCS_REF=custom_ref`

I can rephrase "Uses the value in LIB_VCS_REF if specified or main otherwise" since I guess it's not completely clear